### ISSUE TITLE
Publish docker image on ghcr.io?

### ISSUE THREAD

**Issue by sollie** [martinlindhe/eqbc-go#1]
Hi,

Would you consider also publishing your docker images on Github Container Registry, ghcr.io?
I regularly have issues with rate limiting on docker hub.

It's pretty straight forward.
You need a secret with `repo, write:packages` (not sure if delete:packages is also needed) and something like this:

```yaml
      -
        name: Login to GitHub Container Registry
        uses: docker/login-action@v1
        with:
          registry: ghcr.io
          username: ${{ github.repository_owner }}
          password: ${{ secrets.CR_PAT }}
```
and where you push your image you'd do something like:
```yaml
      -
        name: Build and push
        id: docker_build
        uses: docker/build-push-action@v2
        with:
          push: true
          tags: |
            martinlindhe/eqbc-go:latest
            ghcr.io/martinlindhe/eqbc-go:latest
```

**Comment by martinlindhe** [comment]
Good idea, I'll check it out

**Comment by martinlindhe** [comment]
@sollie I pushed a release to the github container place. Could you help me by verifying you can install and use it ?

Implemented in 0.3.5

**Comment by sollie** [comment]
Thanks!

Seems to behave like the one from docker hub so far. I'm in the middle of setting up my environment from scratch so haven't actually had any traffic on it yet.
